kore/src/Kore/AST/Common.hs

```diff
@@ -1193,6 +1355,8 @@ data Pattern level domain variable child where
         :: !(Top level child) -> Pattern level domain variable child
     VariablePattern
         :: !(variable level) -> Pattern level domain variable child
+    SortPattern
```

Comment:
I would prefer to call this `InhPattern` or `InhabitantPattern` based on its usage in the simplifier.

Can the argument of `\inh` be any pattern?

Reply:
Yes. `\inh` can take any pattern as arguments.

Reply:
Yes I agree. `InhPattern` would be a proper name. I'll fix it now.